composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "myparcelcom/api-sdk",
+  "name": "outgive-inc/api-sdk",
```

Comment:
Hi @outgive-inc ,
Thank you for your contribution! FYI, we will make a pull request with the change bellow (`is_object(...`) because we cannot change the organization name in composer.json.